sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -120,55 +84,76 @@ export interface ModelConfigurationSetting {
     value?: string;
 }
 
-// @public
-export type ModelCreateOrUpdateConfigurationSettingResponse = ModelConfigurationSetting & {
-    _response: coreHttp.HttpResponse & {
-        bodyAsText: string;
-        parsedBody: ModelConfigurationSetting;
-    };
-};
+// @public (undocumented)
+export interface ConfigurationSettingOptions extends Omit<AppConfigurationPutKeyValueOptionalParams, 'label' | 'entity'> {
```

Comment:
I don't think we can use `Omit` since that was added in TypeScript [3.5](https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/#the-omit-helper-type), but our guidelines state we need to support a minimum TypeScript version of [3.1](https://azure.github.io/azure-sdk/typescript_design.html#ts-support-ts).

You should be able to use `Pick` and `Exclude` though.

Reply:
Oops, swapped over to Pick+Exclude